book/07-git-tools/sections/revision-selection.asc

```diff
@@ -180,6 +180,22 @@ Running `git show HEAD@{2.months.ago}` will show you the matching commit only if
 If you have a UNIX or Linux background, you can think of the reflog as Git's version of shell history, which emphasizes that what's there is clearly relevant only for you and your ``session'', and has nothing to do with anyone else who might be working on the same machine.
 ====
 
+[NOTE]
+.Escaping brackets in PowerShell
+====
+
+When using PowerShell on Windows,  brackets like `{` and `}` are special characters and must be escaped.
```

Comment:
```suggestion
When using PowerShell on Windows, brackets like `{` and `}` are special characters and must be escaped.
```

Fix whitespace, I made an error when making the suggestion. My bad! :disappointed: